Modules/Brand/Http/Services/BrandService.php

```diff
@@ -715,15 +874,89 @@ public function getOrderReviews(Request $request): array
                 }
```

Comment:
**Avoid query in the foreach loop.**

if ($orderReviews->isNotEmpty()) {
    $userIds = $orderReviews->pluck('user_id')->toArray();

    $retailers = Retailer::whereIn('user_id', $userIds)
        ->get(['user_id', 'store_name'])
        ->keyBy('user_id');

    foreach ($orderReviews as $orderReview) {
        $retailer = $retailers[$orderReview->user_id] ?? null;

        if ($retailer) {
            $reviews[] = [
                'store_name' => $retailer->store_name,
                'rate' => $orderReview->rate,
            ];
        }
    }
}